### DISCUSSION TITLE
Issue with adding additional email to my profile

### DISCUSSION THREAD

**Discussion by iammarta** [community/community#22656] [New to GitHub]
<p>Hello. I need some help. I am trying to add my additional work email to the current profile <a class="mention" href="/iammarta">@iammarta</a> . But I have this error -&gt; screenshot: <strong>(personal information removed by mods)</strong></p>
<p>The email I am trying to add is not in use, as I have just created it and never used it before.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>If you’re seeing a messaging indicating that email address already exists, it probably does.</p>
<p>It could be that someone previously owned that email address, or that someone made a mistake typing their email address.</p>
<p>You can always try a <a href="https://github.com/password_reset">password reset</a>.</p>
<p>Or you can contact private support, where you can discuss your account details properly.</p>
<p><a href="https://support.github.com/contact" class="onebox" target="_blank">https://support.github.com/contact</a></p>
<p>Please use the email address you are trying to add if you contact support - that will help them verify that you own it.</p>

**Comment by iammarta** [comment]
<p>Great, thanks a lot. The problem is resolved.</p>

**Comment by alifalgibran** [comment]
Hi, I had the same issue but can not resolve. I tried to add my work email to my personal github account but this error appears 

`Error adding (myemail): email is already in use`

**Comment by kavaka123** [comment]
Even I have the same issue, can someone please help me?

**Comment by caiodelgadonew** [comment]
Same issue there @canuckjacq , the email is not shown there but yet I cant add it as a secondary email

**Reply by caiodelgadonew** [reply]
```Error adding <redacted> : email is already in use```

But I have only one email registered and its not this one. pls help

**Reply by canuckjacq** [reply]
Please refer to the accepted answer and try doing a password reset using the email you want to add. You'l have to log out of your account first.

**Reply by caiodelgadonew** [reply]
There was no account on that e-mail. but I've fixed it by creating an account and then deleting it 😢
